Sources/SwiftDocC/Model/Rendering/Variants/VariantCollection+Variant.swift

```diff
@@ -32,11 +32,20 @@ public extension VariantCollection {
         /// Returns a new variant collection containing the traits of this variant collection with the values transformed by the given closure.
         public func mapPatch<TransformedValue>(
             _ transform: (Value) -> TransformedValue
-        ) -> VariantCollection<TransformedValue>.Variant<TransformedValue> {
-            VariantCollection<TransformedValue>.Variant<TransformedValue>(
+        ) -> VariantCollection<TransformedValue>.Variant {
+            VariantCollection<TransformedValue>.Variant(
                 traits: traits,
                 patch: patch.map { patchOperation in patchOperation.map(transform) }
             )
         }
     }
 }
+
+extension VariantCollection.Variant: Equatable where Value: Equatable {
+    public static func == (lhs: VariantCollection<Value>.Variant, rhs: VariantCollection<Value>.Variant) -> Bool {
+        guard lhs.traits == rhs.traits else { return false }
+        guard lhs.patch == rhs.patch else { return false }
+        
+        return true
```

Comment:
Nit: this one seems simple enough that it might be easier to follow like this:

```suggestion
        return lhs.traits == rhs.traits && lhs.patch == rhs.patch
```